### DISCUSSION TITLE
JSON errors

### DISCUSSION THREAD

**Discussion by ghost** [Universal-Team/Universal-Updater#108] [Q&A] [answered]
How do I fix this: https://github.com/GeTuck/UniHB/edit/main/Builds/UniHB-WIP.unistore
I went onto a JSON checker and I got a bunch of stuff wrong. I'm not really sure how to fix these please help.

**Comment by Epicpkmn11** [comment]
You're missing an opening `{` between lines 68 and 69 and a closing `}` at the end ([fixed](https://gist.github.com/Epicpkmn11/3a963566c71134dfa3fa2002e8fb4245)), ([diff](https://gist.github.com/Epicpkmn11/3a963566c71134dfa3fa2002e8fb4245/revisions#diff-aecab446471d70a12425c5689ef6f4ba3d7484a1a0e7bd3a285480e07817620e))

**Comment by SuperSaiyajinStackZ** [comment, marked as answer]
Also the way how you handle the UniStore is not how it's meant to be. With the way you handle it right now, people are supposed to download it manually every single time a new release / change happens. You'd be better to just have a static UniStore file in the repository which you do the changes on, have the UniStore file URL point to that static UniStore file and update the `revision` part on every change you do. That way Universal-Updater *can* check, if there is an update available when loading the UniStore and that's how it's supposed to be. If you really want that `WIP` Unistore to test your changes on first, you can also still do that, and when you think it's good, you apply your `WIP` changes to the static one. Also please don't open several discussions just for a JSON / UniStore error, you could instead just ask on the discord server, if you need help or keep that in one discussion, though the discord server is the better option to be honest.
